packages/rx_bloc_cli/mason_templates/bricks/feature_qr_scanner/__brick__/lib/feature_qr_scanner/util/translate_error_util.dart

```diff
@@ -0,0 +1,8 @@
+{{> licence.dart }}
+
+class TranslateErrorUtil {
```

Comment:
Please keep the showcase features as simple as possible

Reply:
Removed